frontend/packages/dev-console/src/components/AddPage.tsx

```diff
@@ -34,29 +34,25 @@ const handleProjectCreate = (project: K8sResourceKind) =>
 
 const EmptyStateLoader: React.FC<EmptyStateLoaderProps> = ({ resources, loaded, loadError }) => {
   const [noWorkloads, setNoWorkloads] = React.useState(false);
-  const knativeData = _.get(resources, ['knativeService', 'data'], null);
+  const daemonSets = _.get(resources, ['daemonSets', 'data']);
+  const deploymentConfigs = _.get(resources, ['deploymentConfigs', 'data']);
+  const deployments = _.get(resources, ['deployments', 'data']);
+  const statefulSets = _.get(resources, ['statefulSets', 'data']);
+  const knativeService = _.get(resources, ['knativeService', 'data']);
```

Comment:
@jerolimov we are moving away from lodash `_.get` and now prefer to use optional chaining.

Reply:
LOL, we added the same review comments.

Reply:
Ah great. I would also prefer this, but want to keep code similar to the old one. I will change this! Thanks for that hint.